src/model_repository_manager.cc

```diff
@@ -1089,8 +1089,13 @@ ModelRepositoryManager::InitializeModelInfo(
       unmodified = false;
 
       const std::string& override_config = override_parameter->ValueString();
-      RETURN_IF_ERROR(JsonToModelConfig(
-          override_config, 1 /* config_version */, &linfo->model_config_));
+      auto err = JsonToModelConfig(
+          override_config, 1 /* config_version */, &linfo->model_config_);
+      if (!err.IsOk()) {
+        return Status(
+            Status::Code::INVALID_ARG,
+            "Unrecognized config override: " + std::string(err.Message()));
```

Comment:
Not sure if unrecognized is the best word, since this is recognized. Maybe "invalid"? We want the user to know that the config was found but was invalid. Whatever word is clearest for you.

Reply:
"invalid" sounds good, updated!